src/contexts/ValidationsFieldsContext/index.js

```diff
@@ -57,8 +57,14 @@ export const ValidationFieldsProvider = ({ children }) => {
     }
   }
 
-  const loadOriginalValidationFields = async () => {
+  const loadOriginalValidationFields = async (forceLoading) => {
```

Comment:
use an object for params

ex: `const loadOriginalValidationFields = async ({ forceLoading })`